Shared/Coordinators/SettingsCoordinator.swift

```diff
@@ -40,6 +40,9 @@ final class SettingsCoordinator: NavigationCoordinatable {
     @Route(.push)
     var videoPlayerSettings = makeVideoPlayerSettings
 
+    @Route(.push)
+    var filterDrawerButtonSelector = makefilterDrawerButtonSelector
```

Comment:
For future reference, I like these with some sorting. As you can see in the section above, they are sorted alphabetically. The `DEBUG` is split because it's obviously a special case.

Reply:
Re-ordered to fit the original alphabetical layout. This is in the iOS section alphabetically.